lib/constants/styles/widget_styles.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+
+const String DmMono = 'DMMono_regular';
+const String DmSans = 'DMSans_regular';
+const String ClashDisplay = 'Clash Display';
+const defaultPadding = 16.0;
```

Comment:
is this used somewhere?

Reply:
Yes

Reply:
I did a search in the project, it's not used anywhere :D It can be removed